doc8.ini

```diff
@@ -0,0 +1,2 @@
+[doc8]
+path=/docs/source 
```

Comment:
Should this be `docs/source` instead ? (no leading `/`) ? Or `./docs/source` (which is the same) .

Travis CI shows errors and you can see it also inspects CHANGELOG.rst and some other files which are not in this directory.

Reply:
It's the correct syntax. CHANGELOG.rst is referenced in /docs/changelog.rst and therefore checked by doc8. I'll add the file to ignored list.

Reply:
It is OK if we are referencing this file, don't ignore it. However what about these:

```
/home/travis/build/kiwitcms/Kiwi/tcms/templates/email/post_run_save/email.txt
/home/travis/build/kiwitcms/Kiwi/tcms/node_modules/datatables.net-bs/License.txt
```

it looks like doc8 just finds all txt files and checks them, which is not what we want.

Reply:
Added extension parameter and excluded txt folders which have referenced files, as doc8 by default checks for .rst and .txt files.